tools/pd-api-bench/cases/cases.go

```diff
@@ -140,22 +142,23 @@ func newMinResolvedTS() *minResolvedTS {
 }
 
 type minResolvedTSStruct struct {
```

Comment:
You can add `Unary` for `minResolvedTSStruct` to test gRPC

Reply:
We don't support minResolvedTS grpc for now, just extend the api